model/project_vars.go

```diff
@@ -212,35 +203,28 @@ func (projectVars *ProjectVars) FindAndModify(varsToDelete []string) (*adb.Chang
 	)
 }
 
-func (projectVars *ProjectVars) GetRestrictedVars() map[string]string {
-	restrictedVars := map[string]string{}
-
+func (projectVars *ProjectVars) GetVars(t *task.Task) map[string]string {
+	vars := map[string]string{}
+	isAdmin := projectVars.ShouldGetAdminOnlyVars(t)
 	for k, v := range projectVars.Vars {
-		if projectVars.RestrictedVars[k] {
-			restrictedVars[k] = v
+		if !projectVars.AdminOnlyVars[k] || isAdmin {
+			vars[k] = v
 		}
 	}
-	return restrictedVars
+	return vars
 }
 
-// GetAdminOnlyVars currently returns admin only variables for mainline commits and project admins.
-// Will change to GetVars on EVG-16045 after removing restricted vars.
-func (projectVars *ProjectVars) GetAdminOnlyVars(t *task.Task) map[string]string {
-	adminOnlyVars := map[string]string{}
+func (projectVars *ProjectVars) ShouldGetAdminOnlyVars(t *task.Task) bool {
 	if utility.StringSliceContains(evergreen.SystemVersionRequesterTypes, t.Requester) {
-		for k, v := range projectVars.Vars {
-			if projectVars.AdminOnlyVars[k] {
-				adminOnlyVars[k] = v
-			}
-		}
+		return true
 	} else if t.ActivatedBy != "" {
```

Comment:
Now that we're returning, it's cleaner to return more frequently and get things out of if statements. So by checking `if t.ActivatedBy == "" { return false}` at line 220, everything after that can now be out of the else if statement, if that makes sense

Reply:
done